src/test/java/io/pakland/mdas/githubstats/domain/service/GetOrganizationFromIdTest.java

```diff
@@ -1,17 +1,20 @@
 package io.pakland.mdas.githubstats.domain.service;
 
 import io.pakland.mdas.githubstats.application.GetOrganizationFromId;
+import io.pakland.mdas.githubstats.application.exceptions.OrganizationNotFound;
 import io.pakland.mdas.githubstats.domain.Organization;
 import io.pakland.mdas.githubstats.domain.repository.OrganizationRepository;
+import lombok.SneakyThrows;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
 import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class GetOrganizationFromIdTest {
+    @SneakyThrows
```

Comment:
Can we use the "throws" on the method signature instead of the @SneakyThrows lombok anotation? I find that explicitly declaring the different exceptions in the method signature is more self-documenting for the test case.

Reply:
Sure. Makes more sense. I opted for the `SneakyThrows` as it was the first thing suggested by the IDE.